transactions/scripts/get_nft_view.cdc

```diff
@@ -0,0 +1,111 @@
+import ExampleNFT from "../../contracts/ExampleNFT.cdc"
+import MetadataViews from "../../contracts/MetadataViews.cdc"
+
+pub struct NFTView {
```

Comment:
Is is necessary to define a new NFTView in here? Can't we just return the one from the view?

Reply:
ah, that makes sense